apps/chronicles/screens/components/components/SelectMenuScreen.tsx

```diff
@@ -0,0 +1,103 @@
+        { title: "Mixture for Making Water Slightly Wet", value: "slightlyWetWater" },
+        { title: "Mixture for Turning Wine into Water", value: "wineIntoWater" },
+    ];
+
+    const ingredients = [
+        { title: "Frog's Tears", value: "frogsTear" },
+        { title: "Lion's Whiskers", value: "lionsWhiskers" },
+        { title: "Hair of Wolf", value: "wolfHair" },
+        { title: "Phoenix Feather", value: "phoenixFeather" },
+        { title: "Redcap Fungus", value: "redcapFungus" },
+        { title: "Bat Drool", value: "batDrool" },
+        { title: "Snail Shell", value: "snailShell" },
+        { title: "Cow's Tail", value: "cowsTail" },
+    ];
+
+    return (
+        <ScrollView
+            contentInsetAdjustmentBehavior="automatic"
+            contentContainerStyle={styles.container}
+        >
+            <Typography variant="h3">Potion Brewery Lab</Typography>
+
+            <Spacer amount="small" />
+
+            <Typography>
+                Welcome to the Potion Brewery Lab! Here, you can concoct powerful elixirs using
+                ancient recipes and mystical ingredients. With use of the SelectMenu component, you
+                can choose a single potion or multiple ingrediens from a list.
+            </Typography>
+
+            <Spacer />
+
+            <Typography group="paragraph" variant="caption" color="textTertiary">
+                This is a standard select menu
+            </Typography>
+            <Typography>Select your potion:</Typography>
+            <Spacer amount="small" />
+
+            <SelectMenu
+                placeholder="Select here..."
+                items={potions}
+                selectedItem={selectSingleItem}
+                onSelect={setSelectSingleItem}
+            />
+
+            <Spacer />
+            <Spacer />
+
+            <Typography group="paragraph" variant="caption" color="textTertiary">
+                This a mulitselect menu
+            </Typography>
+            <Typography>Select ingrediens:</Typography>
+            <Spacer amount="small" />
+
+            <SelectMenu.Multiselect
+                placeholder="Select here..."
+                items={ingredients}
+                selectedItems={selectMultipleItems}
+                onSelect={setSelectMultipleItems}
+            />
+
+            <Spacer />
+            <Spacer />
+
+            <Typography group="paragraph" variant="caption" color="textTertiary">
+                This is select menu with the «disabled» prop
+            </Typography>
+            <Spacer amount="small" />
+
+            <SelectMenu
+                placeholder="Select here..."
+                items={potions}
+                selectedItem={selectDisabledItem}
+                onSelect={setSelectDisabledItem}
+                disabled
+            />
+        </ScrollView>
+    );
+};
+
+const themeStyles = EDSStyleSheet.create(theme => ({
+    container: {
+        height: "100%",
```

Comment:
It's related to web. 

without 100%:
<img width="1691" alt="Skjermbilde 2024-02-14 kl  10 02 53" src="https://github.com/equinor/mad/assets/83707387/9611de71-9704-4e58-83d7-2aca31407a99">

with 100%:
<img width="1694" alt="Skjermbilde 2024-02-14 kl  10 03 54" src="https://github.com/equinor/mad/assets/83707387/98a50ed4-6c95-435e-a67f-b10ef7e50ba3">

Reply:
Should i just remove it ?

Reply:
Is this also a problem in other screens ? 
If yes, I think it should be removed, just so that it is similar to the other screens.
If no, just keep it 👍